virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,70 @@
-|
-
-|
-| xref:../virt/virtual_machines/virt-manage-vmis.adoc#virt-manage-vmis[Managing virtual machines instances]
-| xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virt-managing-vm-snapshots[Creating and managing virtual machine snapshots]
-|
-
-|
-| xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Controlling virtual machine states]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vm-consoles[Accessing the virtual machine consoles]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[Pass configuration data to virtual machines using secrets, configuration maps, and service accounts]
-|
-|
-
-|
-|===
+You can install and configure a basic {VirtProductName} environment in order to explore its features and functionality.
+
+[NOTE]
+====
+Cluster configuration procedures require `cluster-admin` privileges.
+====
+
+[id="virt-getting-started-before-you-begin"]
+== Before you begin
+
+* Review the xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[installation requirements].
+* Review the xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[storage features] required for cloning, snapshots, and live migration. See xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider] for details.
+* Install the xref:../virt/install/installing-virt-web.adoc#virt-installing-virt-operator_installing-virt-web[{VirtProductName} Operator] and the xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[`virtctl` tool].
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[RHEL virtual machine] using the wizard.
+* Create a Windows virtual machine:
+** Create and customize link:https://cloud.redhat.com/blog/virtual-machines-as-code-with-openshift-gitops-and-openshift-virtualization[Windows boot sources].
+** Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Windows virtual machine] using the wizard.
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers and the QEMU guest agent] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a  xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[a serial console], or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-vnc-console_virt-accessing-vm-consoles[VNC], or in the {product-title} web console.
```

Comment:
Typos. Change to this:

`* Connect to a virtual machine using a xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[serial console] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-vnc-console_virt-accessing-vm-consoles[VNC onsole].`

We can probably remove the part about web console -- it's a lot of consoles in one sentence.

Reply:
Done.